bin/ream/src/cli/lean_node.rs

```diff
@@ -22,6 +22,22 @@ pub struct LeanNodeConfig {
   )]
     pub network: Arc<LeanNetworkSpec>,
 
+    #[arg(
+        long,
+        help = "The directory for storing application data. If used together with --ephemeral, new child directory will be created."
+    )]
+    pub data_dir: Option<PathBuf>,
+
+    #[arg(
+        long,
+        short,
+        help = "Use new data directory, located in OS temporary directory. If used together with --data-dir, new directory will be created there instead."
+    )]
+    pub ephemeral: bool,
+
+    #[arg(long, help = "Purges the database.")]
+    pub purge_db: bool,
```

Comment:
move these flags to the main ReamConfig, as the datadir is for Ream remove them from BeaconNodeConfig as well

Reply:
This is also how geth operates for example